unity-renderer/Assets/Scripts/MainScripts/DCL/CRDTProtocol/CRDTUtils.cs

```diff
@@ -7,7 +7,7 @@ public static long KeyFromIds(int entityId, int componentId)
             // we create an int64 that store both entityId and componentId.
             // entityId on the left 32 bit and componentId in the remaining 
             // 32 bits of the right
-            return (long)entityId << 32 | (long)componentId;
+            return (long)entityId << 32 | (uint)componentId;
```

Comment:
because in some situations (I run a test generating random keys) it was creating a negative `componentId`.
but it doesn't really matter because we are going to change the crdt keys since ts is not very friendly about int64